cmake/ReplicatePythonSourceTree.cmake

```diff
@@ -12,5 +12,6 @@ file(COPY setup.py setup_sub.py pyne DESTINATION "${CMAKE_ARGV3}"
                     PATTERN "*.pyi"
                     PATTERN "*.pxd"
                     PATTERN "*.mas12"
```

Comment:
If we delete the 2012 file, we can probably remove this line.

Reply:
Removed.